mtags/src/main/scala-3/scala/meta/internal/pc/Signatures.scala

```diff
@@ -64,6 +84,12 @@ object ShortName {
     ShortName(sym.name, sym)
 }
 
+case class PrettyType(name: String) extends Type {
+  def hash: Int = 0
+  def computeHash(bind: Binders) = hash
+  override def toString = name
+}
```

Comment:
It would be best to probably just print the types on the go, but that is rather complex. Anyway, looks like a good idea, I will see if I can use it here.

Reply:
Cool but anyway - feel free to ignore that. It's more a thing of taste and as it works for scala2 it shouldn't introduce any significant problem.

Reply:
Ok, let's leave it for now, we might need to work on it anyway in the future.